src/test/ui-fulldeps/auxiliary/roman-numerals.rs

```diff
@@ -10,13 +10,14 @@
 #![feature(plugin_registrar, rustc_private)]
```

Comment:
Maybe let's remove these tests? They are pretty annoying...

Reply:
These are smoke tests for syntactic plugins, the only thing making sure that they don't break.
I think they should exist until we remove the plugins themselves.